README.md

```diff
@@ -30,292 +31,4 @@ var constructorio = new ConstructorIOClient({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `url` | string | ✔ |
-| `pod_id` | string | ✔ |
-| `num_results_viewed` | number | ✔ |
-| `result_count` | integer | |
-| `result_page` | integer | |
-| `result_id` | string | |
-| `section` | string | |
-
-#### Send recommendation click event
-```javascript
-constructorio.tracker.trackRecommendationClick({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `pod_id` | string | ✔ |
-| `strategy_id` | string | ✔ |
-| `item_id` | string | ✔ |
-| `result_count` | integer | |
-| `result_page` | integer | |
-| `result_id` | string | |
-| `section` | string | |
-| `variation_id` | string | |
-| `num_results_per_page` | integer | |
-| `result_position_on_page` | integer | |
-
-#### Send browse results load event
-```javascript
-constructorio.tracker.trackBrowseResultsLoaded({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `url` | string | ✔ |
-| `filter_name` | string | ✔ |
-| `filter_value` | string | ✔ |
-| `section` | string | |
-| `result_id` | string | |
-| `result_count` | integer | |
-| `result_page` | integer | |
-| `selected_filters` | string or array | |
-| `sort_by` | string | |
-| `sort_order` | string | |
-| `items` | array | |
-
-#### Send browse result click event
-```javascript
-constructorio.tracker.trackBrowseResultClick({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `item_id` | string | ✔ |
-| `filter_name` | string | ✔ |
-| `filter_value` | string | ✔ |
-| `variation_id` | string | |
-| `section` | string | |
-| `result_id` | string | |
-| `result_count` | integer | |
-| `result_page` | integer | |
-| `result_position_on_page` | integer | |
-| `num_results_per_page` | integer | |
-| `selected_filters` | string or array | |
-
-#### Receive status of tracking requests
-The status of tracking requests can be observed through emitted events. `messageType` must be either "success" or "error", and the `callback` parameter must be a function that will be passed the message data as the first and only argument.
-```javascript
-constructorio.tracker.on(messageType, callback);
-```
-
-#### Dispatched events
-The `search`, `browse` and `recommendations` module may dispatch custom events on `window` with response data when a request has been completed. The event name follows the following structure: `cio.client.[moduleName].[methodName].completed`. Example consuming code can be found below:
-
-```javascript
-window.addEventListener('cio.client.search.getSearchResults.completed', (event) => {
-  // `event.detail` will be response data
-}, false);
-```
-
-## Development / npm commands
-
-```bash
-npm run lint          # run lint on source code and tests
-npm run test          # run tests
-npm run coverage      # run tests and serves coverage reports from localhost:8081
-npm run docs          # build and serve documentation from localhost:8082
```

Comment:
We should leave these in the readme as well

Reply:
Sounds good 👍